stories/topics/proc-azure-provisioning-aap.adoc

```diff
@@ -34,6 +34,11 @@ Keep this resource group isolated from other resource groups, including the _Res
 . Click btn:[Next].
 . Click the *Business continuity* tab.
 . From the *Disaster Recovery* list, select an option to enable or disable disaster recovery.
+. Select the *Deployment* tab.
+. Note the following requirements in the description:
+** To use {PlatformNameShort}, you must have a valid subscription linked to your Red Hat account.
```

Comment:
To make the docs consistent with the new tab, we might want to add one more bullet here to mention Red Hat account.